<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pracownikWindow</class>
 <widget class="QMainWindow" name="pracownikWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wypożyczalnia samochodów - widok pracownika</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>941</width>
      <height>501</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_listaWyp">
     <attribute name="title">
      <string>Lista wypożyczeń</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>921</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_listWyp">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_listWyp">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_listWyp_oaow">
           <item>
            <widget class="QPushButton" name="pushButton_oddany">
             <property name="text">
              <string>Oddany</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_anulujwyp">
             <property name="text">
              <string>Anuluj wypożyczenie</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_listWyp_odswiez">
             <property name="text">
              <string>Odśwież</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_listWyp_wyloguj">
             <property name="text">
              <string>Wyloguj się</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_listWyp">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_listaWyp"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_formWyp">
     <attribute name="title">
      <string>Formularz wypożyczeń</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_form">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_form_imie">
         <item>
          <widget class="QLabel" name="label_form_imie">
           <property name="text">
            <string>Imie:         </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_form_imie"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_form_nazwisko">
         <item>
          <widget class="QLabel" name="label_form_nazwisko">
           <property name="text">
            <string>Nazwisko:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_form_nazwisko"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_form_email">
         <item>
          <widget class="QLabel" name="label_form_email">
           <property name="text">
            <string>E-mail:     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_form_email"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_form_tel">
         <item>
          <widget class="QLabel" name="label_form_tel">
           <property name="text">
            <string>Nr. tel.:     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_form_tel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pesel">
         <item>
          <widget class="QLabel" name="label_form_pesel">
           <property name="text">
            <string>Pesel:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_form_pesel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_form_dataOd">
         <item>
          <widget class="QLabel" name="label_form_dataOd">
           <property name="text">
            <string>Data od:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_form_dataOd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_form_dataDo">
         <item>
          <widget class="QLabel" name="label_form_dataDo">
           <property name="text">
            <string>Data do:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_form_dataDo"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="listWidget_form">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_form">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>291</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_form_poka">
         <property name="text">
          <string>Pokaż samochody</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_form_wypozycz">
           <property name="text">
            <string>Wypożycz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_form_wyloguj">
           <property name="text">
            <string>Wyloguj się</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Samochody">
     <attribute name="title">
      <string>Samochody</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_sam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>291</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_sam">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_marka">
         <item>
          <widget class="QLabel" name="label_sam_marka">
           <property name="text">
            <string>Marka:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_sam_marka"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_model">
         <item>
          <widget class="QLabel" name="label_sam_model">
           <property name="text">
            <string>Model:  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sam_model"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_rodzaj">
         <item>
          <widget class="QLabel" name="label_sam_rodzaj">
           <property name="text">
            <string>Rodzaj:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_sam_rodzaj"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_nrrej">
         <item>
          <widget class="QLabel" name="label_sam_nrrej">
           <property name="text">
            <string>Nr rej. :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sam_nrrej"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_vin">
         <item>
          <widget class="QLabel" name="label_sam_vin">
           <property name="text">
            <string>Nr VIN:  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sam_vin"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_cena">
         <item>
          <widget class="QLabel" name="label_sam_cena">
           <property name="text">
            <string>Cena:     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sam_cena"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_sam_oddzial">
         <item>
          <widget class="QLabel" name="label_sam_oddzial">
           <property name="text">
            <string>Oddział:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_sam_oddzial"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>291</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_sam_pu">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_sam_pu">
         <item>
          <widget class="QPushButton" name="pushButton_sam_poka">
           <property name="text">
            <string>Pokaż samochody</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_sam_wd">
         <item>
          <widget class="QPushButton" name="pushButton_sam_dodaj">
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sam_wyloguj">
           <property name="text">
            <string>Wyloguj się</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_sam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_pracownicy">
     <attribute name="title">
      <string>Pracownicy</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_pracownicy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>350</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_pracownicy_duwo">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_du">
         <item>
          <widget class="QPushButton" name="pushButton_pracownicy_usun">
           <property name="text">
            <string>Usuń</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pracownicy_dodaj">
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_wo">
         <item>
          <widget class="QPushButton" name="pushButton_pracownicy_odswiez">
           <property name="text">
            <string>Odśwież</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pracownicy_wyloguj">
           <property name="text">
            <string>Wyloguj się</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>291</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_pracownicy">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_imie">
         <item>
          <widget class="QLabel" name="label_pracownicy_imie">
           <property name="text">
            <string>Imię:         </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pracownicy_imie"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_nazwisko">
         <item>
          <widget class="QLabel" name="label_pracownicy_nazwisko">
           <property name="text">
            <string>Nazwisko:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pracownicy_nazwisko"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_email">
         <item>
          <widget class="QLabel" name="label_pracownicy_email">
           <property name="text">
            <string>E-mail:     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pracownicy_email"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_praconwicy_oddzial">
         <item>
          <widget class="QLabel" name="label_pracownicy_oddzial">
           <property name="text">
            <string>Oddział:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_pracownicy_oddzial"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_przelozony">
         <item>
          <widget class="QLabel" name="label_pracownicy_przelozony">
           <property name="text">
            <string>Przełożony:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_pracownicy_przlozony"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_pracownicy_stanowisko">
         <item>
          <widget class="QLabel" name="label_pracownicy_stanowisko">
           <property name="text">
            <string>Stanowisko:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_pacownicy_stanowisko"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout__pracownicy_haslo">
         <item>
          <widget class="QLabel" name="label_pacownicy_haslo">
           <property name="text">
            <string>Hasło:       </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pracownicy_haslo">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_pracownicy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_statystyki">
     <attribute name="title">
      <string>Statystyki</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_statystyki">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_ilosc_wypozyczen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>561</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ilosc_wyp_mies">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>561</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ilosc_wyp_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>561</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ilosc_wyp_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>561</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ilosc_wyp_rej">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>571</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ilosc_wyp_nrej">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>551</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>321</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_statystyki">
       <item>
        <widget class="QComboBox" name="comboBox_statystyki_miesiac"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_statystyki_odswiez">
         <property name="text">
          <string>Odśwież</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_statystyki_wyloguj">
         <property name="text">
          <string>Wyloguj się</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_zaczynamy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zaczynamy">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>571</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zaczynamy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zaczynamy_data">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
